core/templates/dev/head/pages/exploration_player/ConversationSkinDirective.js

```diff
@@ -595,7 +593,7 @@ oppia.directive('conversationSkin', [
                           hasContinueButton: true
                         });
                       }
-
+                      $scope.$broadcast('newCardAvailable', true);
```

Comment:
Pull newCardAvailable into an oppia.constant and use that constant elsewhere. This avoids issues due to typos.

Reply:
done